src/components/Floating/FloatingButton.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from 'next/image';
+import { twMerge } from 'tailwind-merge';
+
+type FloatingButtonProps = {
+  type: 'top' | 'ask';
+  onClick?: () => void;
+};
+
+export function FloatingButton({ type, onClick }: FloatingButtonProps) {
+  const handleClick = () => {
+    if (type === 'top') {
+      window.scrollTo({ top: 0, behavior: 'smooth' });
+    } else if (type === 'ask' && onClick) {
+      onClick();
+    }
+  };
+
+  const iconSrc = type === 'top' ? '/icons/floatVector.svg' : '/icons/floatChat.svg';
+  const iconAlt = type === 'top' ? 'Top Icon' : 'Chat Icon';
+  const iconSize = type === 'top' ? 32 : 35;
+
+  return (
+    <button
+      type="button"
+      onClick={handleClick}
+      className={twMerge(
+        'group fixed bottom-4 flex h-[7.6rem] w-[7.6rem] items-center justify-center rounded-full shadow-lg focus:outline-none',
+        'bg-white text-primary-500 hover:bg-primary-500 hover:text-white',
+      )}
+    >
+      <div className="flex flex-col items-center justify-center">
+        <Image
+          src={iconSrc}
+          alt={iconAlt}
```

Comment:
```suggestion
          alt={`${type} Icon`}
```
P5: 이러면 불필요한 삼항연산자를 안 쓸 수 있습니다!